tests/integration/components/frost-multi-select-test.js

```diff
@@ -457,6 +457,59 @@ describeComponent(...integration('frost-multi-select'), function () {
         expect(onFocus.callCount, 'onFocus is called').to.equal(1)
       })
 
+      // FIX ME
```

Comment:
most of this file just has things moved around. These two click tests was what I tried to get working with the new qualifiers but the item click doesn't seem to work.